libs/dh/market-participant/feature-edit-actor/src/lib/market-participant-edit-actor/dh-market-participant-edit-actor.component.scss

```diff
@@ -15,6 +15,11 @@
  * limitations under the License.
  */
 
+.tab-container {
+  max-height: 680px;
+  overflow-y: auto;
+}
+
```

Comment:
issue: this ends up creating a double scroll (one on the tabs container and one on the whole app). Why is it necessary to set `max-height`?

![image](https://user-images.githubusercontent.com/1096332/170195674-d34e98be-acc1-4744-ae4a-31cd2dc17183.png)

Reply:
I can remove the max-height but then the cancel and save buttons will be pushed out of view. I tried fixing it with flex-box, so that our view would take up remaining visible space, but none of the outer containers have any height restrictions, so they just keep expanding. If you know of a better fix, I'm all ears :D

Reply:
fixed. letting page expand @dzhavat